src/OpenSpaceNet.cpp

```diff
@@ -308,13 +319,23 @@ void OpenSpaceNet::initFilter()
             for (const auto& filterFile : filterAction.second) {
                 FeatureSet filter(filterFile);
                 for (auto& layer : filter) {
-                    auto transform = TransformationChain { std::move(layer.spatialReference().to(imageSr)),
-                                                           std::move(image_->pixelToProj().inverse())};
+                    auto pixelToProj = dynamic_cast<const TransformationChain&>(*pixelToLL_);
```

Comment:
`imageSr` is no longer used.. it's just obtained when it is needed on 328 below.  Please clean up the unused variable.